pegasus/cache/i18n/cs-CZ.yml

```diff
@@ -1378,3 +1378,11 @@
   infinity_shortcode: inf
   playlab_shortcode: hrát
   artist_shortcode: art
+  playlab_choose_theme: Choose your Play Lab theme
```

Comment:
Well I can't not include them, and I gave you a heads up all the real changes were in 1 commit. :P